llvm/lib/Bitcode/Reader/BitcodeReader.cpp

```diff
@@ -3917,6 +3935,13 @@ Error BitcodeReader::parseGlobalVarRecord(ArrayRef<uint64_t> Record) {
     NewGV->setSanitizerMetadata(Meta);
   }
 
+  if (Record.size() > 17 && Record[17]) {
```

Comment:
this won't work if we add another GV record and we have a GV that doesn't specify an explicit code model. we need to distinguish between an invalid record value and 0

Reply:
sorry, I missed the `&& Record[17]`, this is fine